service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceOperations.java

```diff
@@ -141,9 +144,94 @@ public static HttpResponse request(
     }
 
     /**
-     * Sets headers to be used on next HTTP request
-     * @param httpHeaders non null and non empty custom headers
-     * @throws IllegalArgumentException This exception is thrown if headers were null or empty
+     * Send a http request to the IoTHub using the Twin/Method standard, and return its response.
+     *
+     * @param iotHubConnectionString is the connection string for the IoTHub.
+     * @param url is the Twin URL for the device ID.
+     * @param method is the HTTP method (GET, POST, DELETE, PATCH, PUT).
+     * @param payload is the array of bytes that contains the payload.
+     * @param requestId is an unique number that identify the request.
+     * @param connectTimeout the http connect timeout to use.
```

Comment:
Add a comment about this timeout values being in seconds

Reply:
milliseconds, but yeah I'll add a note